### DISCUSSION TITLE
Custom Permissions for Custom Recipes

### DISCUSSION THREAD

**Discussion by Skizzles** [Zrips/CMI#4113] [Feature Requests]
**Description of feature/improvement request:**
Add the ability to create a permission node for custom recipes
---
### Detailed explanation of what it should do, when and in what situation: ###
When you create a custom recipe, the GUI should have a spot where you can enter a custom permission node to be used to be able to craft that recipe.
### Basic explanation as to why this feature is needed and/or useful to have: ###
Overall this would greatly improve the custom recipes so that you could make them unlockable at certain times, areas, quests and really integrate the recipes with other plugins.

**Comment by CloudeLecaw** [comment]
duplicate of this
https://github.com/Zrips/CMI/issues/300 and also of this https://github.com/Zrips/CMI/issues/791

**Comment by impmallet** [comment]
Would like to bump this one because zrips has closed both of the "duplicate" tickets with no explanation.

People paid good money for this plugin, it would be courteous to at least add an explanation for the closure reason. Technical limitation, already working on it under another ticket, just don't want to? Please let us know, it's a slap in the face to just have it closed.

Anyway, this is a huge deal for me and I would love to see this feature added.

Thank you.
